### DISCUSSION TITLE
Database Undefined Column error

### DISCUSSION THREAD

**Discussion by Chad-ll** [Kitware/CDash#2438] [Q&A] [answered]
Hi --

I'm back with another new problem with my newly installed CDash 3.5.0 server (RHEL 8.10, PHP 8.2, PostgreSQL 15). When my user went to create a new project, we got the following error in the `cdash.log` file:
```
production.ERROR: SQLSTATE[42703]: Undefined column: 7 ERROR: column "ldapfilter" of relation "project" does not exist
LINE 1: ...us", "emailmaxitems", "emailmaxchars", "imageid", "ldapfilte...
                                                             ^ (Connection: pgsql, SQL: insert into "project" (xxxxxxxxxxxxxx) values (PublicProject, A testing playground for basic submissions........
```
Then it goes on for a bit. (The "xxxxxxxx" part is a list of all of the columns I think.)

I'm not a "DB person" by any means :-) but the `php artisan db` commands seem happy. Although `php artisan db:seed` does error with 
```
In Response.php line 391:
Undefined array key "project"
```

Thanks,
Chad

**Comment by williamjallen** [comment, marked as answer]
It looks like you haven't run the migrations for 3.5 somehow.  What does `php artisan migrate` report?

As an aside, `php artisan db:seed` is for development purposes, and will overwrite any existing data.

**Reply by Chad-ll** [reply]
Well, that certainly did something!

It did about 10 migrations, the last one being `ldap_project_membership` and that seems to have made CDash happy enough to let us create a project. Yay! 

I would swear I ran that migration though. Hmmm. 

(The `php artisan db:seed` was me totally grasping for straws earlier.)

Thank you again for your help! Hopefully my users will be able to do their thing with CDash on this network now.
